bats_test/upload-file/create-file.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
-yes "10 chunk test"  | head --bytes 2621440 > /home/node/10Chunks.txt
+yes "10 chunk test"  | head -c 2621440 > /home/node/10Chunks.txt
```

Comment:
Is `-c` portable across OSes? I know --bytes doesn't work on OSX but I'm not sure if -c works on Linux (I may have that backwards, but double check either way).

Reply:
Not sure about Windows, but -c works on inside our Linux Docker environment and on MacOS. In hindsight, this change was unnecessary. I was just having local compatibility issues there